native/cocos/application/ApplicationManager.cpp

```diff
@@ -50,9 +56,14 @@ ApplicationManager::ApplicationPtr ApplicationManager::getCurrentAppSafe() const
 }
 } // namespace cc
 
-//
+void cocos_close() { // NOLINT(readability-identifier-naming)
+    // Called in the platform layer, because the platform layer is isolated from the application layer
+    // It is the platform layer to drive applications and reclaim resources.
+    cc::ApplicationManager::getInstance()->closeAllApplications();
```

Comment:
Can do it in `cocos_destory()`?

Reply:
The entire exit process needs to be refactored, so here is the simple treatment

Reply:
What's the problem of exit process?

Reply:
The initialization process and exit process of the entire engine need to be carefully designed, and it is still confusing now.

Reply:
Yes, the whole exit needs to be refactored. Calling cocos_close is to call engine.close(). Here, exit(0) will be called; basically, the logic behind will not be called. This was originally done this way, and I've kept the operation.

Reply:
> The initialization process and exit process of the entire engine need to be carefully designed, and it is still confusing now.

Ok. Then it is better not export more functions.

Reply:
@qiuguohua please create an issue to refactor the init and exit process.